### ISSUE TITLE
Combining different filtering providers to validate lookups

### ISSUE THREAD

**Issue by cdhorn** [DNSCrypt/dnscrypt-proxy#724]
It would be nice to have a mode/plugin where the lookup is validated against a list of separate DNS filtering services. Some may be better at filtering malware and phishing sites (NortonConnectSafe/Quad9), others pornography (CleanBrowsing/SafeDNS), others possibly advertising (AdGuard). The idea would be to query them all in parallel and require a valid response from all of them. NavyTitanium wrote some code in Python that does this, as per his DNSMasterChef repository, but native support for such functionality in dnscrypt-proxy would simplify things a good bit.
